lib/pages/demo.dart

```diff
@@ -405,7 +405,6 @@ class _GalleryDemoPageState extends State<GalleryDemoPage>
       );
     } else {
       section = AnimatedSize(
-        vsync: this,
```

Comment:
Would this class still need to be a `TickerProviderStateMixin`?

Reply:
This is still needed because `_GalleryDemoPageState` is also creating an `AnimationController` that uses a `TickerProvider`